client/src/App.js

```diff
@@ -13,31 +13,46 @@ function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
   const [userEmail, setEmail] = useState(null);
+  const [info, setInfo] = useState([]);
 
-  useEffect(() => {
-    axios.get('http://localhost:3000/auth/isLoggedIn', {withCredentials: true})
-    .then((result) => {
+  useEffect(async () => {
+    await axios.get('http://localhost:3000/auth/isLoggedIn', {withCredentials: true})
+    .then( async (result) => {
       console.log('is login auth:', result.data)
-      setIsLoggedIn(result.data);
       setIsLoading(false);
+      if (result.data) {
+        console.log('is login auth:', result.data)
+        setIsLoggedIn(result.data.loggedIn);
+        setEmail(result.data.info)
+        await axios.get('http://localhost:3000/todoList/info',{ params: { email: result.data.info } })
+        .then((response) => {
+          console.log('info response:', response.data.results[0])
+          setInfo(response.data.results[0]);
+        })
+        .catch((err) => {
+          console.log('info err:', err);
+          return err;
+        })
+      }
     })
     .catch((err) => {
       console.log(err);
+      return err;
     })
   }, [isLoggedIn])
```

Comment:
i recently learned that we can do something like this here: 
  }, [isLoggedIn, userEmail]);
  
 Am not sure if it's applicable here, but just thought i'd mention!